### PR TITLE
Test and fix for mysql bookinfo

### LINKED ISSUE #1075: bookinfo mysql error with new images
```
module.js:327
    throw err;
    ^

Error: Cannot find module 'mysql'
```
Seems like #1014 or #1022 broke the ratings v2 image, needs to be rebuilt/reverted

### PR DESCRIPTION
**Pull request by ldemailly**
Fixes #1075

Allows to have both mongoldb and mysql coexisting in same test.

Note: had to rebuild the image, into gcr.io as I don’t have access to
the other. (Initially tried to just use an older image but we also
changed the protocol so that didn’t work…)
Added mysql as a dependency to package.json

Ps: as mixer tests mongodb maybe we can have this only test mysql

```release-note
Add bookinfo mysql to regression tests. Fix the ratings sample app so it doesn't throw when talking to mysql
```

### CODE REVIEW

File: tests/e2e/tests/bookinfo/demo_test.go [hunk 1]
```diff
@@ -423,6 +432,24 @@ func TestDbRouting(t *testing.T) {
 		fmt.Sprintf("Success! Response matches with expected! %s", respExpr), t)
 }
 
+func TestDbRoutingMysql(t *testing.T) {
+	var err error
+	var rules = []string{testMysqlRule}
```
Comment (ldemailly) [inline tests/e2e/tests/bookinfo/demo_test.go]: copy-pasta from TestDbRouting with just the rule changed

File: samples/bookinfo/kube/route-rule-ratings-mysql.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+kind: RouteRule
+metadata:
+  name: ratings-test-v2-mysql
+spec:
+  destination:
+    name: ratings
+  precedence: 3
+  route:
+  - labels:
+      version: v2-mysql
```
Comment (ldemailly) [inline samples/bookinfo/kube/route-rule-ratings-mysql.yaml]: same as route-rule-ratings-db.yaml with a -mysql added to select the mysql one and they can coexist

**Comment by saumoh** [comment]
Note: Not specific to this change.
The test for db's (mongo and mysql) which checks for number of glyphon-star is not going to find the errors in failure to access the db backend. There are following options to address this:
- Either there should be a version of bookinfo output that can be distinguishable as text output or
- the total number of glyphon-stars should be != 10 or 
- The names of the 'reviewer's' should be picked up from the db and not statically set by review's micro service.

**Comment by ldemailly** [comment]
it is publicly accessible but can/will be pushed to docker for consistency 

I agree that it only test the backend if the rule works which isn’t great (thus the TODO I added last time around)

**Comment by ldemailly** [comment]
can someone put a / in front ot the lgtm :) tia!

**Pull request merged**